discovery-client.js

```diff
@@ -101,6 +102,7 @@ let createOrUpdateApiInternal = async function(curlUrl, token, bodyContent, meth
                 Authorization: 'Bearer ' + token,
                 Accept: 'application/json',
                 'Content-Type': contentType,
+                'Record-Api-Version': RECORD_API_VERSION,
```

Comment:
Based on the naming change discussion this should change to `x-ibm-record-api-version`